modules/virt-about-liveness-readiness-probes.adoc

```diff
@@ -6,7 +6,14 @@
 
 = About liveness and readiness probes
 
-_Liveness probes_ determine if a virtual machine instance (VMI) is still responding. If the probe fails, the VMI is stopped. A replacement VMI is then created to restore virtual machine responsiveness.
+Use liveness and readiness probes to detect and handle unhealthy virtual machine instances (VMIs). You can include one or more probes in the specification of the VMI to ensure that traffic does not reach a VMI that is not ready for it and that a new instance is created when a VMI becomes unresponsive.
 
-_Readiness probes_ tell services and endpoints that the virtual machine instance
-is ready to receive traffic from services. If readiness probes fail, the virtual machine instance is removed from applicable endpoints until the probe recovers.
+A _readiness probe_ determines whether a VMI is ready to accept service requests. If the probe fails, the VMI is removed from the list of available endpoints until the probe recovers.
```

Comment:
Nit of all nits (brace yourself for pettiness): 
Since we say 'liveness and readiness probes' in title and para and examples, should liveness come before readiness here?

Reply:
Done